pkg/controller/trafficrouting/trafficrouting_controller.go

```diff
@@ -124,8 +125,8 @@ func (r *TrafficRoutingReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	case v1alpha1.TrafficRoutingPhaseFinalizing:
 		done, err = r.trafficRoutingManager.FinalisingTrafficRouting(newTrafficRoutingContext(tr), false)
 		if done {
-			newStatus.Phase = v1alpha1.TrafficRoutingPhaseHealthy
-			newStatus.Message = "TrafficRouting is Healthy"
+			newStatus.Phase = v1alpha1.TrafficRoutingPhaseInitial
+			newStatus.Message = "TrafficRouting is Initializing"
```

Comment:
why changing routing to initializing?

Reply:
Changing the status to Healthy works fine for gateway and ingress since they only check if the resource exists when initializing, however, for custom network providers, the original configuration is stored in annotation in initialization, and the annotation will be removed after finalising, without this annotation, customController can't do EnsureRoutes(). So it is necessary to first initialize after finalising, or next time customController will not work.